.travis.yml

```
@@ -0,0 +1,15 @@
+sudo: false
+language: node_js
+node_js: node
```

```suggestion
node_js: lts/*
```
This restricts us to the latest long-term-stable version of node, rather than the latest released version. (This is what broke the Travis build for CLUE at one point.)